stacks-signer/src/signerdb.rs

```diff
@@ -826,6 +826,16 @@ impl SignerDb {
         try_deserialize(result)
     }
 
+    /// Return the count of globally signed blocks in a tenure (identified by its consensus hash)
+    pub fn get_globally_signed_block_count_in_tenure(
+        &self,
+        tenure: &ConsensusHash,
+    ) -> Result<i64, DBError> {
+        let query = "SELECT COUNT(*) FROM blocks WHERE consensus_hash = ?1 AND state = ?2 AND signed_over = 1";
```

Comment:
Do we need to include `signed_over = 1` here? I believe that indicates if this signer signed, but I think we don't care whether this signer signed or not, as long as it is globally accepted.